prow/config/jobs/proxy-1.10.yaml

```diff
@@ -0,0 +1,231 @@
+branches:
+- release-1.10
+image: gcr.io/istio-testing/build-tools-proxy:release-1.10-2021-04-02T23-15-48
+jobs:
+- command:
+  - ./prow/proxy-presubmit.sh
+  name: test
+  node_selector:
+    testing: build-pool
+  requirements:
+  - cache
+  - gcp
+  timeout: 4h0m0s
+  types:
+  - presubmit
+- command:
+  - ./prow/proxy-presubmit-asan.sh
+  name: test-asan
+  node_selector:
+    testing: build-pool
+  requirements:
+  - cache
+  - gcp
+  timeout: 4h0m0s
+  types:
+  - presubmit
+- command:
+  - ./prow/proxy-presubmit-tsan.sh
+  name: test-tsan
+  node_selector:
+    testing: build-pool
+  requirements:
+  - cache
+  - gcp
+  timeout: 4h0m0s
+  types:
+  - presubmit
+- command:
+  - ./prow/proxy-presubmit-release.sh
+  name: release-test
+  node_selector:
+    testing: build-pool
+  requirements:
+  - cache
+  - gcp
+  timeout: 4h0m0s
+  types:
+  - presubmit
+- command:
+  - ./prow/proxy-presubmit-centos-release.sh
+  image: gcr.io/istio-testing/build-tools-centos:master-2021-04-02T23-15-48
+  modifiers:
+  - optional
+  name: release-centos-test
+  node_selector:
+    testing: build-pool
+  requirements:
+  - cache
+  - gcp
+  timeout: 4h0m0s
+  types:
+  - presubmit
+- command:
+  - entrypoint
+  - ./prow/proxy-presubmit-wasm.sh
+  name: check-wasm
+  node_selector:
+    testing: build-pool
+  requirements:
+  - cache
+  - gcp
+  - docker
+  timeout: 4h0m0s
+  types:
+  - presubmit
+- command:
+  - entrypoint
+  - ./prow/proxy-postsubmit.sh
+  name: release
+  node_selector:
+    testing: build-pool
+  requirements:
+  - cache
+  - gcp
+  - docker
+  timeout: 4h0m0s
+  types:
+  - postsubmit
+- command:
+  - ./prow/proxy-postsubmit-centos.sh
+  image: gcr.io/istio-testing/build-tools-centos:master-2021-04-02T23-15-48
+  name: release-centos
+  node_selector:
+    testing: build-pool
+  requirements:
+  - cache
+  - gcp
+  timeout: 4h0m0s
+  types:
+  - postsubmit
+- command:
+  - ../test-infra/tools/automator/automator.sh
+  - --org=istio
+  - --repo=istio
+  - '--title=Automator: update proxy@$AUTOMATOR_SRC_BRANCH in $AUTOMATOR_ORG/$AUTOMATOR_REPO@$AUTOMATOR_BRANCH'
+  - --labels=auto-merge,release-notes-none
+  - --modifier=update_proxy_dep
+  - --token-path=/etc/github-token/oauth
+  - --cmd=bin/update_proxy.sh $AUTOMATOR_SHA
+  image: gcr.io/istio-testing/build-tools:master-2021-04-02T23-15-48
```

Comment:
this seems wrong?

Reply:
Changed all build-tools references to release-1.10